api/applications/applicationRouter.js

```diff
@@ -320,183 +170,67 @@ router.post(
- *                $ref: '#/components/schemas/Application'
- *              example:
- *                application_notes: 'Example of the application_notes'
- *      '401':
- *        $ref: '#/components/responses/UnauthorizedError'
- */
-
-// updates applicants application_notes field
-
-router.put(
-  '/update-notes/:id',
-  authRequired,
-  adminRequired,
-  (req, res, next) => {
-    const application_id = req.params.id;
-    const application_notes = req.body.application_notes;
-    Application.updateApplicationNotes(application_id, application_notes)
-      .then((response) => {
-        if (response == 0)
-          return next({ message: 'Application does not exist', status: 404 });
-
-        res.status(200).json(response[0]);
+      .then((result) => {
+        res.send({ status: result.status, message: result.data });
       })
-      .catch(next);
-  }
-);
-
-/**
- * @swagger
- * /application/intake/:role/:id}:
- *  put:
- *    summary: Gets mentor or mentee intake data using role and id
- *    description: Mentor or mentee intake data is retrieved through a get request, where role and profile_id are sent as params. The mentor or mentee correlating with the profile_id will be returned on a successful get or a 404 if the profile does not exist. The authRequired and adminRequired middleware functions keep this endpoint secure.
- *    tags:
- *      - application
- *    security:
- *      - okta: [authRequired, adminRequired]
- *    parameters:
- *      - in: param
- *        name: role / profile id
- *        schema:
- *          type: string
- *        description: Profile ID of pending applicant and role (Mentors or Mentees)
- *    responses:
- *      '200':
- *        description: Response from successful get
- *        content:
- *          application/json:
- *            schema:
- *              type: object
- *              properties:
- *                message:
- *                  description: Result of request as object
- *                  type: object
- *              items:
- *                $ref: '#/components/schemas/Application'
- *              example:
- *                message: {"result": [{"profile_id": "4F177Y8xr4Ap1q0y","first_name": "Raiden","last_name": "Nelson","email": "fake@email.com","city": "Ashland","state": "Oregon","country": "USA",
-"formerly_incarcerated": true,"underrepresented_group": true,"low_income": true,"list_convictions": ["Infraction"],"subject": "General Programming","experience_level": "Expert","job_help": false,"industry_knowledge": false,"pair_programming": false,"other_info": "Notes"}]} 
- *      '401':
- *        $ref: '#/components/responses/UnauthorizedError'
- */
-
-// gets mentor or mentee application intake data from Data Science
-
-router.get(
-  '/intake/:role/:id',
-  authRequired,
-  adminRequired,
-  (req, res, next) => {
-    const profile_id = req.params.id;
-    const role = req.params.role;
+      .catch((err) => {
+        next({ status: res.status, message: err });
```

Comment:
I'm not sure if this is correct: wouldn't res still be empty at this point? Shouldn't the status being sent back to FE be defined by BE (IE. 500)? Has this been tested? It looks like status would end up being undefined. Let me know if I'm wrong

Reply:
this was a woopsie on my part. this should be err.status, and err.message. I've fixed these accordingly--thanks for noticing the oddity.